gapic/src/main/com/google/gapid/models/Experiments.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.models;
+
+public class Experiments {
+  public static final boolean ShouldEnableExperiments = false;
+
+  boolean shouldDisableAF;
+
+  public Experiments() {
```

Comment:
Make this class immutable. All fields can be `public final`.

Reply:
Even though it looks simpler with one experiment in it, I think that would make the use of this class very ugly after a few experiment added. Anytime we change an experiment settings, we would need to create new Experiments and copy all the other fields from the existing one which would end up with lots of unnecessary code. I think having this as a final but a mutable class is much cleaner.